docs/reference/module-options.md

```diff
@@ -372,3 +372,457 @@ module.exports = {
+        _url: 1,
+        publishedAt: 1
+      }
+    }
+  },
+  // ...
+}
+```
+
+### `piecesFilters`
+
+<!-- TODO: Link to a guide on using piece filters when available. -->
+<!-- TODO: Link to a better query builder guide when available. -->
+`piecesFilters` can be configured as an array of objects to support filtering pieces on an [index page](/reference/glossary.md#index-page). Each object must have a `name` property associated with a valid [query builder](/guide/major-changes.md#queries). These include:
+
+- Custom query builders configured in an app that include a `launder` method
+- Field names whose field types automatically get builders:
+  - `boolean`
+  - `checkboxes`
+  - `date`
+  - `float`
+  - `integer`
+  - `relationship`
+  - `select`
+  - `slug`
+  - `string`
+  - `url`
+
+When the index page is served, configured filters will be represented on a `req.data.piecesFilters` object (`data.piecesFilters` in the template). If you include `counts: true` in a filter object, the number of pieces matching that filter are included on `req.data.piecesFilters` properties.
+
+#### Example
+
+```javascript
+// modules/article-page/index.js
+// 👆
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    piecesFilters: [
+      { name: '_author' },
+      {
+        name: 'category',
+        counts: true
+      }
+    ]
+  },
+  fields: {
+    _author: {
+      type: 'relationship',
+      label: 'Author'
+    },
+    category: {
+      type: 'select',
+      label: 'Category',
+      choices: [
+        // Category choices here
+      ]
+    }
+    // Other fields...
+  },
+  // ...
+}
+```
+
+### `pieceModuleName`
+
+Piece page types are each associated with a single piece type. If named with the pattern, `[piece name]-page`, the associated piece type will be detected automatically. For example, if the `article-page` module extends `@apostrophecms/piece-page-type`, it will automatically be associated with an `article` piece type.
+
+This pattern can be overridden or ignored by explicitly setting `pieceModuleName` to an active piece type name. Ths can be useful if there is more than one piece page type for a single piece type (e.g., to support different functionality in each).
+
+#### Example
+
+```javascript
+// modules/team-page/index.js
+// 👆 This module name would look for a piece type named `team` if not for
+// `pieceModuleName`
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    pieceModuleName: 'person'
+  },
+  // Maybe there's code here to group people by team.
+  // ...
+}
+```
+
+### `previous`
+
+If set to `true`, Apostrophe will include the previous piece, based on the [sort option](#sort), on `req.data.previous` when serving a [show page](/reference/glossary.md#show-page). This is useful to add links to a show page directing visitors to the previous item in a series (e.g., the next newest blog post). If not set, `req.data.previous` will not be available.
```

Comment:
"This is useful to add links to a show page directing visitors to the previous item in a series (e.g., the next newest blog post). If not set, `req.data.previous` will not be available."

Feels like we should use "pagination" here (single page pagination) to simplify this sentence.

Reply:
How so? It's not part of our normal "pagination" functionality from index pages. I don't want to confuse it with that.